### PR TITLE
Upgrade Kubernetes examples to v1.3.0

### LINKED ISSUE #265: Upgrade Kubernetes cluster to v1.3.0
Upgrade the `k8s` and `k8s-install` (static manifest) to Kubernetes v1.3.0.

### PR DESCRIPTION
**Pull request by dghubble**
* Upgrades `k8s`, `k8s-install`, and `k8s-docker` example clusters to Kubernetes v1.3.0

@pbx0 @aaronlevy 
Closes #265

### CODE REVIEW

**Comment by dghubble** [comment]
Kubernetes v1.3.0 conformance tests against `k8s-install` on a physical cluster.

```
go run hack/e2e.go -v --test --test_args="--ginkgo.focus=\[Conformance\]" --check_version_skew=false
Client Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.0", GitCommit:"283137936a498aed572ee22af6774b6fb6e9fd94", GitTreeState:"clean", BuildDate:"2016-07-17T07:58:14Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.0+coreos.1", GitCommit:"83e9c91279813860f241b68d076d58f9c5871357", GitTreeState:"clean", BuildDate:"2016-07-06T20:04:26Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
```

```
Ran 98 of 324 Specs in 2653.709 seconds
SUCCESS! -- 98 Passed | 0 Failed | 0 Pending | 226 Skipped PASS

Ginkgo ran 1 suite in 44m13.817511523s
Test Suite Passed
2016/07/17 02:24:10 e2e.go:214: Step 'Ginkgo tests' finished in 44m13.927669549s
```

File: examples/ignition/k8s-master.yaml [hunk 1]
```diff
@@ -363,13 +356,12 @@ storage:
                       },
                       {
                         "args": [
-                          "-machines=http://127.0.0.1:4001",
-                          "-addr=0.0.0.0:53",
-                          "-ns-rotate=false",
-                          "-domain=cluster.local."
+                          "--cache-size=1000",
+                          "--no-resolv",
+                          "--server=127.0.0.1#10053"
```
Comment (aaronlevy) [inline examples/ignition/k8s-master.yaml]: Is the "#" intentional?

**Comment by aaronlevy** [comment]
One question, but otherwise lgtm

File: examples/ignition/k8s-master.yaml (hunk 1 above)
Comment (dghubble) [inline examples/ignition/k8s-master.yaml]: Copied from https://github.com/coreos/coreos-kubernetes/blob/master/multi-node/generic/controller-install.sh#L460

Comment (aaronlevy) [inline examples/ignition/k8s-master.yaml]: Yeah, looks like I grabbed it from here too: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in#L91

**Pull request merged**